src/main/java/org/mule/module/apikit/validation/body/form/transformation/MultipartFormDataBinaryParameter.java

```diff
@@ -27,6 +32,25 @@ public MultipartFormDataBinaryParameter(byte[] inputStream, MediaType mediaType)
 
   @Override
   public void validate(Parameter parameter) throws InvalidFormParameterException {
-    //Parsers currently doesn't validate files
+    if (!parameter.getFileProperties().isPresent()) {
+      return;
+    }
+    FileProperties properties = parameter.getFileProperties().get();
+    Set<String> fileTypes = properties.getFileTypes();
+    Integer minValue = properties.getMinLength();
+    Integer maxValue = properties.getMaxLength();
+
+    if (isNotEmpty(fileTypes) && !fileTypes.contains(mediaType.toString())) {
+      throw new InvalidFormParameterException(format("Invalid content type: %s", mediaType.toString()));
```

Comment:
I would like to see if there is any wild card (*) at the rfc before pushing this change,, we would also be sure to talk about this feature at the releas notes since it may "break" some existing api

Reply:
Included case */* https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#file and testing.
Sure, this need to be included in release notes